src/diagonal.works/b6/ingest/worlds.go

```diff
@@ -0,0 +1,43 @@
+package ingest
+
+import (
+	"diagonal.works/b6"
+)
+
+var DefaultWorldFeatureID = b6.FeatureID{
+	Type:      b6.FeatureTypeCollection,
+	Namespace: "diagonal.works/world",
+	Value:     0,
+}
+
+type Worlds interface {
+	FindWorld(id b6.FeatureID) MutableWorld
+}
+
+type MutableWorlds struct {
```

Comment:
maybe this could all be in mutable ?

Reply:
I think I'll keep all the Worlds code together for the moment, and maybe split it out later when we better know the shape.